terraform/modules/stack/iam_role_policy.tf

```diff
@@ -181,3 +181,29 @@ resource "aws_iam_role_policy" "notifier_metrics" {
   policy = data.aws_iam_policy_document.cloudwatch_putmetrics.json
 }
 
+# backfill
+
+resource "aws_iam_role_policy" "bags_tracker_vhs_manifests_backfill_readonly" {
+  role   = module.bags_tracker_backfill.task_role_name
+  policy = var.vhs_manifests_readwrite_policy_backfill
+}
+
+resource "aws_iam_role_policy" "bags_tracker_backfill_metrics" {
+  role   = module.bags_tracker_backfill.task_role_name
+  policy = data.aws_iam_policy_document.cloudwatch_putmetrics.json
+}
+
+resource "aws_iam_role_policy" "bag_register_backfill_replica_primary_readonly" {
+  role   = module.bag_register_backfill.task_role_name
+  policy = data.aws_iam_policy_document.replica_primary_readonly.json
+}
+
+resource "aws_iam_role_policy" "bag_register_backfill_metrics" {
+  role   = module.bag_register_backfill.task_role_name
+  policy = data.aws_iam_policy_document.cloudwatch_putmetrics.json
+}
+
+resource "aws_iam_role_policy" "azure_replicator_replica_primary_readonly" {
```

Comment:
Possibly worth a TODO: delete on this one? The backfill roles we'll remember to clean up because the services will get deleted; this one won't.

Reply:
The idea was to add all of this backfill stuff at the end of each file so that we can just go ahead and delete everything under a `# backfill` comment. I'll add a todo in each one so we know and it's not just in my head